ionic-test-app/.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs
```

Comment:
We can rely on the linters and config on the project root so unnecessary to include this in the test app